extensions/amp-bind/0.1/amp-state.js

```diff
@@ -121,7 +121,7 @@ export class AmpState extends AMP.BaseElement {
   parseAndUpdate() {
     if (this.localData_ === undefined) {
       this.localData_ = this.parse_();
-      if (this.localData_) {
+      if (this.localData_ !== null) {
```

Comment:
Wait, can `tryParseJson` return `undefined`?

Reply:
No, but it can return falsy values e.g. `JSON.parse(0) === 0`.